packages/special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -1,3 +1,4 @@
+import 'preact/devtools'
```

Comment:
Can this be added to dev builds only?

Reply:
oops! yeah let's remove it for now, and then add a esbuild thing to ensure we don't include it by mistake